src/box/replication.cc

```diff
@@ -653,6 +653,11 @@ replicaset_update(struct applier **appliers, int count, bool keep_connect)
 	replicaset.applier.connected = 0;
 	replicaset.applier.loading = 0;
 	replicaset.applier.synced = 0;
+
+	struct applier *appliers_for_delete[VCLOCK_MAX] = {};
+	int appliers_for_delete_count = 0;
+	int csw = fiber()->csw;
```

Comment:
Which method of initialization of such variables is preferable: to wrap `csw` in `ifndef NDEBUG` or just to leave everything as it is.

Excess memory is used in release for a variable, we don't use.  But AFAICS, we don't wrap such things as I proposed in general.

Reply:
I'd leave it as is, no need to worry